### PR TITLE
[feat]tools-v2: add bs query volume clone-recover

### PR DESCRIPTION
**Pull request by baytan0720**
### What problem does this PR solve?

Issue Number: #2584  

Problem Summary:

add bs query volume clone-recover

### What is changed and how it works?

What's Changed:

How it Works:

Usage:
```bash
curve bs query volume clone-recover
```

Output:
```bash
+------+--------------------------------------+--------------------------------------+----------+-------+----------+--------+----------+--------+----------+---------------------+
| USER |                 SRC                  |               TASK ID                | TASKTYPE | FILE  | FILETYPE | ISLAZY | NEXTSTEP | STATUS | PROGRESS |        TIME         |
+------+--------------------------------------+--------------------------------------+----------+-------+----------+--------+----------+--------+----------+---------------------+
| root | a19b5e5e-b306-488f-8e6d-d87282c869cb | d26e27a8-fcbd-4f7a-adf8-53795217cbb0 |  clone   | /root |   file   | false  |   end    |  done  |    1     | 2006-01-02 15:04:05 |
+------+--------------------------------------+--------------------------------------+----------+-------+----------+--------+----------+--------+----------+---------------------+
```

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

File: tools-v2/pkg/cli/command/curvebs/delete/volume/clone/clone.go [hunk 1]
```diff
@@ -138,11 +108,21 @@ func (rCmd *CloneCmd) RunCommand(cmd *cobra.Command, args []string) error {
...
 				}
 			}
 			item[cobrautil.ROW_RESULT] = cobrautil.ROW_VALUE_SUCCESS
-			rCmd.TableNew.Append([]string{item["User"], item["Src"], item["UUID"], item["File"], item["Result"]})
+			rCmd.TableNew.Append([]string{
+				item["User"].(string),
+				item["Src"].(string),
+				item["UUID"].(string),
+				item["File"].(string),
+				item["Result"].(string),
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/delete/volume/clone/clone.go:123]: defined as constant

File: tools-v2/pkg/cli/command/curvebs/query/volume/clone-recover/clone-recover.go [hunk 2]
```diff
@@ -0,0 +1,210 @@
+package clone_recover
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/query/volume/clone-recover/clone-recover.go:1]: move down

**Comment by Cyber-SiKu** [comment]
@baytan0720 pls fix `conflicts`.

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Review by Cyber-SiKu — APPROVED** [review]

**Review by caoxianfei1 — APPROVED** [review]

**Pull request merged**